README.md

```diff
@@ -1,2 +1,10 @@
 # taf-members
 Members site for Tippicanoe Arts Federation.
+
+
+## Instructions to run
+1. Install docker and compose
+2. Run `docker-compose up`. The server should be running on port 8000
+3. To run a database migration: `docker-compose run web python manage.py migrate`
+
```

Comment:
@drewatk To start the server, we still need to run `manage.py runserver`, right? That might be good to add.

I think ultimately we want the whole thing behind Nginx somewhere. But that might just be necessary when deployed, not necessary in docker.

Reply:
check out docker-compose.yml
That command is run by the web service being launched (`docker-compose up`)